### PR TITLE
clarified steps and added how to cancel

### PR DESCRIPTION
**Pull request by lhirata**

### CODE REVIEW

**Comment by sploiselle** [comment]
LGTM

File: training/long-running-queries.md [hunk 1]
```diff
@@ -68,33 +68,43 @@ $ ./ycsb -duration 5m -tolerate-errors -concurrency 2 -rate-limit 100 'postgres:
...
+~~~ sql
+> CANCEL QUERY '14dacc1f9a781e3d0000000000000001';
+~~~
+
+### Step 4. Find Long-Running Queries in the Admin UI
 
 1. Access the Admin UI at `http://localhost:8080`
 
-2. Find your long-running queries from **Dashboards** > **Slow Running Queries**.
+2. Find your long-running queries from **Dashboards** > **Slow Requests**.
```
Comment (jseldess) [inline training/long-running-queries.md]: Do sql queries show up on that dashboard, @sploiselle?

Comment (lhirata) [reply training/long-running-queries.md]: checked with @Amruta-Ranade. long-running queries will display on the SQL dashboard.

Comment (lhirata) [reply training/long-running-queries.md]: (updated accordingly)

Comment (jseldess) [reply training/long-running-queries.md]: But that won't be backported to 1.1, will it, @Amruta-Ranade? These training pages are for 1.1 (our stable release)...

File: training/long-running-queries.md [hunk 2]
```diff
@@ -68,35 +68,37 @@ $ ./ycsb -duration 5m -tolerate-errors -concurrency 2 -rate-limit 100 'postgres:
...
 
+    {% include copy-clipboard.html %}
     ~~~ sql
-    > SELECT query_id, query, start FROM [SHOW CLUSTER QUERIES]
-          WHERE start < (now() - INTERVAL '1 hour');
+    > SELECT * FROM [SHOW CLUSTER QUERIES]
+      WHERE start < (now() - INTERVAL '1 hour');
     ~~~
 
 3. We're done with both the load generator, so you can stop it by going to its terminals and quitting the process.
```
Comment (jseldess) [inline training/long-running-queries.md]: Just noticing that this sentence doesn't quite make sense. @sploiselle, did we mean to say this?

> We're doing with the load generator, so you can stop it by going to its terminal and quitting the process.

Or is there a reason we're referring to "terminals" in the plural?

Comment (lhirata) [reply training/long-running-queries.md]: edited - @sploiselle let me know if it should be changed.

**Pull request merged**